pos_stock_available_online/static/src/js/Screens/ProductScreen/ProductItem.esm.js

```diff
@@ -0,0 +1,43 @@
+/** @odoo-module **/
+
+import ProductItem from "point_of_sale.ProductItem";
+import Registries from "point_of_sale.Registries";
+import {format} from "web.field_utils";
+import utils from "web.utils";
+
+const StockProductItem = (ProductItem) =>
+    class StockProductItem extends ProductItem {
+        format_quantity(quantity) {
+            const unit = this.env.pos.units_by_id[this.props.product.uom_id[0]];
+            var formattedQuantity = `${quantity}`;
+            if (unit) {
```

Comment:
```suggestion
            if (!unit) return `${formattedQuantity}`;
```

Reply:
We should use no-negative condition: https://eslint.org/docs/latest/rules/no-negated-condition

Reply:
nice! there is a eslint about that!